cluster-autoscaler/simulator/predicatechecker/testchecker.go

```diff
@@ -17,11 +17,18 @@ limitations under the License.
 package predicatechecker
 
 import (
+	"k8s.io/autoscaler/cluster-autoscaler/utils/scheduler"
 	clientsetfake "k8s.io/client-go/kubernetes/fake"
 )
 
 // NewTestPredicateChecker builds test version of PredicateChecker.
 func NewTestPredicateChecker() (PredicateChecker, error) {
+
+	cfg, err := scheduler.GetSchedulerConfig("")
```

Comment:
nit: Why not use the default config directly?

Reply:
Let me think about this :pray:

Reply:
This has been updated to use the default scheduler config in case no scheduler config is passed through the function parameter: https://github.com/kubernetes/autoscaler/pull/5708/files#diff-5818325464670a5cd54280fe253b0841a163da5de2764491a3065805785ffb1bR51-R56